html-report/index.html

```diff
@@ -14,5 +14,6 @@
     <div id="root"></div>
     <script type="text/javascript" src="${relative_path}build/app.min.js"></script>
     ${log}
+    <div class="copy content">Powered by Juno</div>
```

Comment:
Or "Generated with ❤️ by Juno"

Like GitHub does when you embed a gist somewhere 

<img width="274" alt="screen shot 2017-06-07 at 00 52 55" src="https://user-images.githubusercontent.com/967132/26853701-b6a732f2-4b1b-11e7-8506-9d4d1f6d3b85.png">